src/kernels/composable_kernel/include/tensor_operation/blockwise_gemm_xdlops.hpp

```diff
@@ -71,6 +71,9 @@ struct BlockwiseGemmBlockABlockBThreadCTransANormalBNormalC_xdlops
         static_assert(BlockSize == GemmMWaves * GemmNWaves * WaveSize,
                       "BlockSize != GemmMWaves * GemmNWaves * WaveSize\n");
 
+        static_assert((MRepeats == 1 && NRepeats == 1) || CK_USE_AMD_XDLOPS_INLINE_ASM == 0,
```

Comment:
This equation should be easy to read and align with your description about assert failure
((mrepeats=1 && nrepeats==1) && inline_asm=1)